nemo/core/classes/modelPT.py

```diff
@@ -407,7 +407,7 @@ def setup_multiple_test_data(self, test_data_config: Union[DictConfig, Dict]):
             if self._test_dl is not None and type(self._test_dl) in [list, tuple]:
                 self._test_names = ['test_{}_'.format(idx) for idx in range(len(self._test_dl))]
 
-    def setup_optimization(self, optim_config: Optional[Union[DictConfig, Dict]] = None):
+    def setup_optimization(self, optim_config: Optional[Union[DictConfig, Dict]] = None, delayed_sched_init=False):
```

Comment:
Can we remove this: `delayed_sched_init`?

Reply:
I can do that. Then, I guess then we are just re-do the LR schedule init when using master optimizer wrapper.
I guess that should be fine(?).

Reply:
Done